src/style.css

```diff
@@ -1 +1,39 @@
-/* add styles here */
+.body {
```

Comment:
You're using the `.body` class to style the body of your document. However, according to the checklist, you should avoid using tag names for styling, except for `html` and `body`. In this case, it would be more appropriate to directly style the `body` tag instead of creating a class for it.